File: buddybridge-frontend/src/app/restrict/base/grupo_acesso/model/grupoacessoDTO.ts
import { AcessoDTO } from "./acessoDTO";

export class GrupoAcessoDTO {
  idGrupoAcesso?: number | undefined;
  descricaoGrupoAcesso?: string;
  ativoGrupoAcesso?: boolean;
  acessos?: AcessoDTO[];

  constructor() {
    this.idGrupoAcesso = undefined;
    this.descricaoGrupoAcesso = '';
    this.ativoGrupoAcesso = true;
    this.acessos = [];
  }
}

File: buddybridge-backend/src/main/java/br/com/buddybridge/core/controleacesso/model/GrupoAcessoDTO.java
package br.com.buddybridge.core.controleacesso.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class GrupoAcessoDTO {

    private Long idGrupoAcesso;
    private String descricaoGrupoAcesso;
    private boolean ativoGrupoAcesso;
    private List<AcessoDTO> acessos;

}
